src/component/metrics/threads-modal.tsx

```diff
@@ -88,8 +88,8 @@ export class ThreadsModal extends React.Component<IThreadsModalProps, IThreadsMo
     }
 
     return (
-      <Modal isOpen={showModal} toggle={handleClose} className="modal-lg">
-        <ModalHeader toggle={handleClose}>Threads dump</ModalHeader>
+      <Modal isOpen={showModal} toggle={() => handleClose()} className="modal-lg">
```

Comment:
same comment as earlier. it would be nice to change `handleClose` to match expected signature

Reply:
sure, this way ?

Reply:
Yes but now you don't need the closures anymore right. So you can revert to original code

Reply:
I still have a compilation error if I use `toggle={handleClose}` even with `handleClose` declared like this:
```
handleClose = e => {
    this.setState({
      showModal: false
    });
  };
```

the error is:
src/component/metrics/threads-modal.tsx:91:33 - error TS2769: No overload matches this call.
  Overload 1 of 2, '(props: ModalProps | Readonly<ModalProps>): Modal', gave the following error.
    Type 'Function' is not assignable to type '((event: MouseEvent<any, MouseEvent>) => void) | ((event: KeyboardEvent<any>) => void)'.
      Type 'Function' is not assignable to type '(event: KeyboardEvent<any>) => void'.
        Type 'Function' provides no match for the signature '(event: KeyboardEvent<any>): void'.
  Overload 2 of 2, '(props: ModalProps, context: any): Modal', gave the following error.
    Type 'Function' is not assignable to type '((event: MouseEvent<any, MouseEvent>) => void) | ((event: KeyboardEvent<any>) => void)'.
      Type 'Function' is not assignable to type '(event: KeyboardEvent<any>) => void'.

Reply:
Ok let me have a look. I think I can fix it